### PR TITLE
[#16] Clean up code

### PR DESCRIPTION
**Pull request by kammanz**
- All cleaned up!!!

### CODE REVIEW

File: src/components/PlayerDetails.js [hunk 1]
```diff
@@ -2,15 +2,14 @@ import React from 'react';
...
-
     if (!player) {
         return <h3>Select a Player</h3>
     }
 
     return (
         <div>
             <h3>Player Stats</h3>            
-            <div>Feild Goal Percentage:{ player.FG_PCT }</div>
+            <div>Field Goal Percentage:{ player.FG_PCT }</div>
```
Comment (Tumne) [inline src/components/PlayerDetails.js]: still isn't done please change `{ player.FG_PCT }` to `{player.FG_PCT}`

File: src/components/PlayersList.js [hunk 2]
```diff
@@ -12,20 +12,20 @@ class PlayersList extends React.Component {
         const { players } = this.props;
         return players.map((player, index) => {
             return (
-                <div key={ index } onClick={() => this.props.selectPlayer(player) }>
+                <div key={ index } onClick={ () => this.props.selectPlayer(player) }>
```
Comment (Tumne) [inline src/components/PlayersList.js]: same here

File: src/components/PlayersList.js [hunk 3]
```diff
@@ -12,20 +12,20 @@ class PlayersList extends React.Component {
...
 
     render() {
-        const renderPlayers = this.renderPlayers();
+        
         console.log(this.props.players);
         return (
             <div>
                 <h3>Players</h3>
-                <div>{ renderPlayers }</div>
+                <div>{ this.renderPlayers() }</div>
```
Comment (Tumne) [inline src/components/PlayersList.js]: same here

File: src/components/PlayersList.js [hunk 4]
```diff
@@ -36,8 +36,7 @@ const getMyState = (state) => {
     return { players: state.players };
 }
 
-export default connect(getMyState, {
-        fetchPlayers: fetchPlayers,
-        selectPlayer: selectPlayer,
-     }
+export default connect(getMyState,
+        { fetchPlayers },
+        { selectPlayer },
```
Comment (Tumne) [inline src/components/PlayersList.js]:
```
{
  fetchPlayers,
  selectPlayer,
}
```

File: src/components/PlayersList.js [hunk 5]
```diff
@@ -12,20 +12,20 @@ class PlayersList extends React.Component {
...
+                <div key={ index } onClick={ () => this.props.selectPlayer(player) }>
                     { player.PLAYER_NAME }
                 </div>
             ); 
         });
     }
 
     render() {
-        const renderPlayers = this.renderPlayers();
+
```
Comment (Tumne) [inline src/components/PlayersList.js]: delete this empty line

**Review by Tumne — CHANGES_REQUESTED** [review]

**Comment by kammanz** [comment]
@Tumne fixed PR ready for review.

**Review by Tumne — APPROVED** [review]

**Pull request merged**